### PR TITLE
Add Platformatic to Organizations

### PR DESCRIPTION
**Pull request by mcollina**
Signed-off-by: Matteo Collina <hello@matteocollina.com>

#### Checklist

- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by kibertoad — APPROVED** [review]

File: src/website/data/benchmarks.yml [hunk 1]
```diff
@@ -2,27 +2,27 @@ reference: 71768
 frameworks:
```
Comment (kibertoad) [inline src/website/data/benchmarks.yml:2]: not sure if intentional, but if this is more up-to-date, then nice

**Review by luisorbaiceta — APPROVED** [review]
🙌

File: src/website/data/benchmarks.yml (hunk 1 above)
Comment (mcollina) [reply src/website/data/benchmarks.yml:2]: not intentional, will fix

Comment (mcollina) [reply src/website/data/benchmarks.yml:2]: anyway, we can also keep those.

**Pull request merged**